jfr-streaming/README.md

```diff
@@ -4,7 +4,48 @@
+| BUFFER_METRICS            | false           | `process.runtime.jvm.buffer.usage`\n`process.runtime.jvm.buffer.limit`\n`process.runtime.jvm.buffer.count`                                                                                                  |
+| CLASS_LOAD_METRICS        | false           | `process.runtime.jvm.classes.current_loaded`\n`process.runtime.jvm.classes.loaded`\n`process.runtime.jvm.classes.unloaded`                                                                                  |
+| CONTEXT_SWITCH_METRICS    | true            | `process.runtime.jvm.cpu.context_switch`                                                                                                                                                                    |
+| CPU_COUNT_METRICS         | true            | `process.runtime.jvm.cpu.limit`                                                                                                                                                                             |
+| CPU_UTILIZATION_METRICS   | false           | `process.runtime.jvm.cpu.utilization`\n`process.runtime.jvm.system.cpu.utilization`                                                                                                                         |
+| GC_DURATION_METRICS       | false           | `process.runtime.jvm.gc.duration`                                                                                                                                                                           |
+| LOCK_METRICS              | true            | `process.runtime.jvm.cpu.longlock`                                                                                                                                                                          |
+| MEMORY_ALLOCATION_METRICS | true            | `process.runtime.jvm.memory.allocation`                                                                                                                                                                     |
+| MEMORY_POOL_METRICS       | false           | `process.runtime.jvm.memory.init`\n`process.runtime.jvm.memory.committed`\n`process.runtime.jvm.memory.usage`\n`process.runtime.jvm.memory.limit`\n``\n``\n`process.runtime.jvm.memory.usage_after_last_gc` |
+| NETWORK_IO_METRICS        | true            | `process.runtime.jvm.network.io`\n`process.runtime.jvm.network.time`                                                                                                                                        |
+| THREAD_METRICS            | false           | `process.runtime.jvm.threads.count`                                                                                                                                                                         |
```

Comment:
I've added descriptive text and this table to the readme to help explain how all the stuff in this PR comes together.